diagnostic.md

```diff
@@ -21,13 +21,21 @@ Write the `curl` request you'd use to retrieve a list of all donuts on the
 server.
 
 ```sh
-# your answer here
+curl --request GET http://10.13.105.206:3000/items
 ```
 
 Write an AJAX request to retrieve a list of all donuts on the server.
 
 ```js
-let getDonuts = /* your answer here */;
+let getDonuts = function(){
+$.ajax({                                    // $ is jquery, exercute ajax in ().  everything inside is an object literal
+url: 'http://10.13.105.206:3000/items',     // a key is url, retrieve a list of items
+type: 'GET'                                 // tell ajax to get
+})                                          // ajax takes these info and use it
+
+.done(logResponseBody)                      // register that function when you hear done
+.fail(logRequestError);                     // register that function when you hear fail
+};
```

Comment:
This is correct, but your indenting makes this very difficult to read. Please be cognizant of it going forward!